app/views/estimate_flow/benefits.html.slim

```diff
@@ -37,7 +37,7 @@
       = render "add_benefit_form",
                form:,
                legend: { text: t(".legend"), size: "l" },
-               hint_text: FeatureFlags.enabled?(:disregard_cost_of_living) ? t(".add_more_hint_with_col") : t(".add_more_hint")
```

Comment:
I think we should remove the "orphaned" strings from en.yml, like `estimate_flow.benefits.add_more_hint` here.